### DISCUSSION TITLE
Tempo Datasource loki search is not getting set

### DISCUSSION THREAD

**Discussion by sandhraprakash** [grafana/grafana#51360] [Data Sources]
I am provisioning tempo datasource as per the [document](https://grafana.com/docs/grafana/latest/datasources/tempo/#provision-the-tempo-data-source). and In Loki data source derived fields are set. 
```
lokiSearch:
        datasourceUid: 'Loki'
```
The above snippet appears to have been set in the UI after running the yaml
![image](https://user-images.githubusercontent.com/34446301/175477616-be53041c-a1ab-44d3-bd12-675e6194736d.png)

Derived fields are working fine. we are able to see the trace by navigating from logs. But when we try to search for traces from logs the following error comes
![image](https://user-images.githubusercontent.com/34446301/175478195-79888aa0-49b4-4347-8452-23ff99ecf952.png)

PS: The issue is not reproducible if we are creating and configuring datasources manually. this happens only in provisioned datasources

**Comment by johnalotoski** [comment]
Did you find a resolution for this?  I'm running into the same issue with provisioned datasources.

**Comment by johnalotoski** [comment]
After reviewing issue https://github.com/grafana/grafana/issues/39562, got it working by making sure to add a datasourceUid for the expected name of the datasource in the provisioning declaration, rather than allow a random string to be picked.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
